src/states/elder/tests.rs

```diff
@@ -64,6 +90,10 @@ struct ElderUnderTest {
 
 impl ElderUnderTest {
     fn new() -> Self {
+        Self::with_min_section_size(DEFAULT_MIN_SECTION_SIZE)
```

Comment:
hmm... so there is other places calling this constructor with different size?

Reply:
Yes, in the `accept_previously_rejected_client_after_reaching_min_section_size` test.